README.md

```diff
@@ -84,20 +93,19 @@ The following properties are accepted:
 
 - `externalNodeModules`
 
-  _Type_: `array<string>`\
+  _Type_: `array<string>`
 
   List of Node modules to include separately inside a node_modules directory.
 
 - `ignoredNodeModules`
 
-  _Type_: `array<string>`\
+  _Type_: `array<string>`
 
   List of Node modules to keep out of the bundle.
 
 - `nodeBundler`
 
-  _Type_: `string`\
-  _Default value_: `zisi`
+  _Type_: `string` _Default value_: `zisi`
```

Comment:
I believe this might have been accidental.

Reply:
It was! Thanks. Fixed in https://github.com/netlify/zip-it-and-ship-it/pull/410/commits/b29fd723a3db7b82cbeb128c1ea70714a441042c.